config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show', as: 'user'
+  get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  get '/users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
```

Comment:
- [ ] It would be nice if you make your routes using the resources method to make your routes in the latest syntax of rails 😊 as you can see below ⬇️

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```